### DISCUSSION TITLE
如何将现有Flink集群中的任务一次性迁移到 StreamPark 中进行纳管？

### DISCUSSION THREAD

**Discussion by cobolbaby** [apache/streampark#3288] [General]
目前我们有五六个Flink集群，每个集群中任务数100+，如果要用 StreamPark 统一进行纳管，是要让所有任务的开发者都通过 StreamPark 再提交一次吗？还是说有什么快速接入的办法？

**Comment by wolfboys** [comment]
只能重新接入，没有其他快捷方式

**Reply by cobolbaby** [reply]
工作量有点巨大．
